docs/testcases/TC10-008-dna.html

```diff
@@ -0,0 +1,229 @@
+					<li class="toc-item"><a class="bg-light" href="#TCdata">Test Data</a></li>
+					<li class="toc-item"><a class="bg-light" href="#TCsteps">Test Steps</a></li>
+				</ul>
+			</li>
+		</ul>
+	</nav>
+	
+	<!-- Main page content -->
+	<main class="container-fluid mb-auto flex-shrink-0" id="main-content">	
+		<div class="row">	
+			<div class="col-md-9 col-xl-8 py-1 px-1 py-md-2 pl-md-3 offset-md-3 offset-xl-2 order-1">
+				<div class="card shadow">
+					<div class="card-body">
+						<h1 class="card-title" id="TCtitle"><!-- InstanceBeginEditable name="TCtitle" -->Input text element - Accessibility tree visibility (<code class="code">visibility:hidden</code>)<!-- InstanceEndEditable --></h1>
+							<h2 class="border-bottom" id="TCsum">Test Case Summary</h2>
+								<h3 id="TCid">Test Case ID</h3>
+									<p><!-- InstanceBeginEditable name="TCid" -->TC10-008-DNA<!-- InstanceEndEditable --></p>
+								<h3 id="TCdesc">Test Case Description</h3>
+									<p><!-- InstanceBeginEditable name="TCdesc" -->The purpose of this test case is to detect  attributes that would contribute to the accessible name and accessible description computation and calculate the text alternative for <code class="code">input</code> elements. The code sample does not present itself via the <a href="https://wicg.github.io/aom/explainer.html">Accessibility tree/Accessibility object model (AOM)</a>, visually, or programmatically  and consequently cannot be identified by the tester within the current test process. Consequently, a successful test should indicate that this test does not apply to Baseline Test 10 Forms.<br>
+                                        <br>
+                                    <strong><u>Note</u>:</strong> If the intent of the developer is to make the <code class="code">input</code> element  available to users as a part of meaningful ICT functionality, then the CSS property <code class="code">visibility:hidden</code> should not be used with interactive elements.<!-- InstanceEndEditable --></p>
+								<h3 id="TCbase">Applicable ICT Baseline</h3>
+									<p><!-- InstanceBeginEditable name="TCbaseURL" --> <a href="https://section508coordinators.github.io/ICTTestingBaseline/10Forms.html">10. Forms</a><!-- InstanceEndEditable --></p>
+									<h4><!-- InstanceBeginEditable name="TCbaseheading4" -->Complete and programmatically associated form labels and instructions<!-- InstanceEndEditable --></h4>
+									<h5>Test Instructions</h5>
+								
+									<!--Test instructions text: //-->
+									<!-- InstanceBeginEditable name="TCbasetext" -->
+									<h6 id="test-procedure-for-131-info-and-relationships-and-412-name-role-value"> 4.1.2 Name, Role, Value and 1.3.1 Info and Relationships</h6>
+									<ol>
+									  <li>Find all form components. Examples include buttons, text fields, radio buttons, checkboxes, read-only fields, and multi-select lists.</li>
```

Comment:
Suggest adding the Identify Content and Test Instructions headings and separating the steps just to stay consistent with the content in the Basleine.

Reply:
I included in test step 1 the first item under "Identify Content" from the Baseline verbatim.  I did not include the second item from the Baseline "Identify content" because the visibility:hidden implementation prevents the second step from being possible.  

Therefore, unsure what would need to be done here to align with the baseline any more than is currently written.

Reply:
Hmmm - It looks to me like you included both of the the Identify Content steps verbatim, but you included them under Test Instructions. I think it's fine to include them. I'm just suggesting that we include them just as they are written in the baseline: separated under different headings, Identify Content and Test Instructions, just to avoid confusion is someone flips back and forth:
![image](https://user-images.githubusercontent.com/29068131/91309977-4e600580-e77f-11ea-8f77-0f3c5708a798.png)

Reply:
So are we saying modify the "Test Steps" section of the template (table) to provide a subsection of "Identify Content" and "Test Instructions"?  That makes sense to me to stay more in alignment with the Baseline but I didn't want to assume that because it would be a departure from previous test cases that I don't remember discussing in our meetings.  Let me know if  this is what you're saying and I'll make the change.  Thx!

Reply:
I guess I'm not sure if I'm saying it is necessary to include the Identify Content section in every test case, but if we do, and we are essentially quoting or copying/pasting from the Baseline, then we should probably still keep the Identify Content steps separate from the Test Instructions steps (in the Applicable Baseline section of the test case).

Reply:
But once we get to the Test Steps (in the table under the Test Case Detail section), we don't need to separate them or identify which steps come from Identify Content or from the Test Instructions sections of the Baseline. At that point, our steps are specific to just this test case. My original comment refers to the quoted section of the Applicable Baseline, not the Test Steps under the Test Case Detail section.

Reply:
08/26/2020: Done with my commits from today.